### PR TITLE
contributor:Updated instructions which were outdated in the closed PR #1296

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
Original PRs: 
* https://github.com/spryker/spryker-docs/pull/1331
* https://github.com/spryker/spryker-docs/pull/1345

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/the-docker-sdk/202204.0/configure-services.md [hunk 1]
```diff
@@ -144,18 +140,17 @@ docker/sdk clean-data
...
-See [MySQL documentation](https://dev.mysql.com/doc/) for more details.
+For more details, see [MySQL documentation](https://dev.mysql.com/doc/).
 
 #### Configure MySQL
 
-Follow the steps below to switch database engine to MySQL:
+To switch the database engine to MySQL, follow these steps: 
 
-1. Adjust `deploy.*.yaml` in the `services:` section:
+1. In the `services:` section, adjust `deploy.*.yaml`:
```
Comment (andriitserkovnyi) [inline docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:153]: The `services:` section is in `deploy.*.yaml`, not the other way around

File: docs/scos/dev/the-docker-sdk/202204.0/configure-services.md [hunk 2]
```diff
@@ -179,15 +174,15 @@ docker/sdk up --build --data
...
-See [PostgreSQL documentation](https://www.postgresql.org/docs/) for more details.
+For more details., see [PostgreSQL documentation](https://www.postgresql.org/docs/).
 
 #### Configure PostgreSQL
 
-Follow the steps below to switch database engine to PostgreSQL:
+To switch the database engine to PostgreSQL, follow these steps:
 
-1. Adjust `deploy.*.yml` in the `services:` section:
+1. In the `services:` section, adjust `deploy.*.yml`:
```
Comment (andriitserkovnyi) [inline docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:185]: same

File: docs/scos/dev/the-docker-sdk/202204.0/configure-services.md [hunk 3]
```diff
@@ -212,14 +207,14 @@ docker/sdk up --build --data
...
 
-* [Configure Elasticsearch](/docs/scos/dev/back-end-development/data-manipulation/data-interaction/search/configure-elasticsearch.html) to learn more about ElastciSearch configuration in Spryker.
-* [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html) for more information on ElasticSearch.
+* [Configure Elasticsearch](/docs/scos/dev/back-end-development/data-manipulation/data-interaction/search/configure-elasticsearch.html)—describes ElastciSearch configuration in Spryker.
+* [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html)—provides detailed information about ElasticSearch.
 
 ### Configure ElasticSearch
 
-Adjust `deploy.*.yml` in the `services:` section to open the port used for accessing ElasticSearch:
+In the `services:` section, adjust `deploy.*.yml` to open the port used for accessing ElasticSearch:
```
Comment (andriitserkovnyi) [inline docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:217]: same

File: docs/scos/dev/the-docker-sdk/202204.0/configure-services.md [hunk 4]
```diff
@@ -231,17 +226,17 @@ services:
...
 
 In Docker SDK, Kibana UI is provided as a service by default.
 
 ### Configure Kibana UI
 
-Follow the steps to configure an endpoint for Kibana UI:
+To configure an endpoint for Kibana UI, follow these steps:
 
-1. Adjust `deploy.*.yml` in the `services:` section:
+1. In the `services:` section, adjust `deploy.*.yml`:
```
Comment (andriitserkovnyi) [inline docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:239]: same

File: docs/scos/dev/the-docker-sdk/202204.0/configure-services.md (hunk 3 above)
Comment (andriitserkovnyi) [reply docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:217]: revert the reset as well

File: docs/scos/dev/the-docker-sdk/202204.0/configure-services.md [hunk 5]
```diff
@@ -454,89 +447,138 @@ BLACKFIRE_CLIENT_ID={client_id} BLACKFIRE_CLIENT_TOKEN={client-token} docker/sdk
...
+    tag: spryker/php:7.4 # the image tag that has been previously used in `image`
     php:
         ...
         enabled-extensions:
             ...
             - newrelic
 ```
 
-#### Alternative configuration
+Once New Relic has been enabled in your deploy file, [contact Spryker Support](/docs/scos/user/intro-to-spryker/support/how-to-contact-spryker-support.html) to have it integrated into your environment. This can be done by submitting an infrastructure change request through the [Support Portal](/docs/scos/user/intro-to-spryker/support/how-to-use-the-support-portal.html). The Support team will confirm that a New Relic APM account is available for you and ensure that the correct application naming convention is set up to cascade to the appropriate APM.
```
Comment (andriitserkovnyi) [inline docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:494]: Do I need to push and deploy the updated file before contacting support?

Comment (ajspryker) [reply docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:494]: As I do not have access to push to any branches within spryker/spryker-docs, I have made this change to my initial PR #1331.

Comment (andriitserkovnyi) [reply docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:494]: I'm asking about the instructions in the doc, not the PR :D 

Meaning, do I need to push and deploy the updated deploy file before contacting support?

File: docs/scos/dev/the-docker-sdk/202204.0/configure-services.md [hunk 6]
```diff
@@ -454,89 +447,138 @@ BLACKFIRE_CLIENT_ID={client_id} BLACKFIRE_CLIENT_TOKEN={client-token} docker/sdk
...
+    {
+        return [
+            new NewRelicMonitoringExtensionPlugin(),
+        ];
+    }
+}
 ```
 
-{% info_block warningBox "Note" %}
+With `new \SprykerEco\Service\NewRelic\Plugin\NewRelicMonitoringExtensionPlugin()` being returned within the `getMonitoringExtensions()` function, this tells the Monitoring class to include New Relic. Without these changes, New Relic only reports the base (for example, `index.php`) without showing the appropriate endpoint or class being called with each transaction.
```
Comment (andriitserkovnyi) [inline docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:561]: How is this related to showing apps as different APMs?

Comment (ajspryker) [reply docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:561]: I updated the language to this in my own PR #1331 with commit [6d2b61698a273d541ad7ae33a8e8598bc3c2a4f0](https://github.com/spryker/spryker-docs/pull/1331/commits/6d2b61698a273d541ad7ae33a8e8598bc3c2a4f0). I am unable to update this PR to show this change.

File: docs/scos/dev/the-docker-sdk/202204.0/configure-services.md (hunk 5 above)
Comment (ajspryker) [reply docs/scos/dev/the-docker-sdk/202204.0/configure-services.md:494]: I have further updated the language in my own PR at commit [9f6b7ebb777fe3f1ef0c0defb5311d3453946a56](https://github.com/spryker/spryker-docs/pull/1331/commits/4c4f3061940f0abba6984d670f6b77e0b671a576).

**Review by andriitserkovnyi — APPROVED** [review]
